cli/chaosli/cmd/root.go

```diff
@@ -101,6 +98,16 @@ func initConfig() {
 	}
 }
 
+func initDDMark() {
```

Comment:
nit: Rather adding a comment `// setup ddmark` below the function name, we can rename the function by setupDDMark.